### PR TITLE
server: failed notify reports failure

### LINKED ISSUE #1914: /notify clientID can be 'too many users' instead of an int
With duplicate servers, server startup fails with:
`WARNING: Server localhost couldn't set client id to: 'too many users'`

which is kind of OK, but points to a problem, `['/done', '/notify', clientID]` seem to put in `'too many users'` as third argument.

### LINKED ISSUE #2328: ServerStatus failOSCFunc shouldn't set clientID
The OSC message sent on failure to register with scsynth for notifications returns a symbol 'too many users', not a clientID.

bug is here: https://github.com/supercollider/supercollider/blob/master/SCClassLibrary/Common/Control/ServerStatus.sc#L58

Stack trace is from atom-supercollider which is rich text and cannot be copy and pasted here. See pretty picture below.

```
Server:newBusAllocators
Args
this    
▶ Server localhost  
Vars
numControl  Integer 16384
numAudio    Integer 112
controlBusOffset    Symbol 'too many users'
audioBusOffset  Symbol 'too many users'
offset  Symbol 'too many users'
n   Integer 1
Server:newAllocators
Args
this    
▶ Server localhost  
Server:clientID_
Args
this    
▶ Server localhost  
val Symbol 'too many users'
a Function defined in undefined:undefined
Args
msg Array [ /fail, /notify, too many users ]
OSCArgsMatcher:value
Args
this    
▶ an OSCArgsMatcher  
testMsg Array [ /fail, /notify, too many users ]
time    Float 2.620803172
addr    
▶ a NetAddr(127.0.0.1, 57110)  
recvPort    Integer 57121
OSCFuncAddrMessageMatcher:value
Args
this    
▶ an OSCFuncAddrMessageMatcher  
msg Array [ /fail, /notify, too many users ]
time    Float 2.620803172
testAddr    
▶ a NetAddr(127.0.0.1, 57110)  
recvPort    Integer 57121
OSCMessageDispatcher:value
Args
this    
▶ an OSCMessageDispatcher  
msg Array [ /fail, /notify, too many users ]
time    Float 2.620803172
addr    
▶ a NetAddr(127.0.0.1, 57110)  
recvPort    Integer 57121
Main:recvOSCmessage
Args
this    a Main
time    Float 2.620803172
replyAddr   
▶ a NetAddr(127.0.0.1, 57110)  
recvPort    Integer 57121
msg Array [ /fail, /notify, too many users ]
```

screen shot 2016-09-07 at 11 19 08

### PR DESCRIPTION
**Pull request by telephon**
The server shouldn’t try to set the clientID on failure.

This fixes #2328. Related to #1914.

This is still a partial solution.

### CODE REVIEW

**Comment by crucialfelix** [comment]
This would be better on 3.8 not on master - this is a bugfix for a feature that was introduced in 3.8

**Comment by telephon** [comment]
Don't we cherry-pick the recent bugfixes in master to make 3.8.1 ? This is not the only one, I think.

**Comment by crucialfelix** [comment]
nope, we are in beta right now. that means that bugs are found and fixed right away. fixes go into beta-2 release

**Comment by telephon** [comment]
I see. That means the fixes are not tested by other devs until they are released and then merged back into master. Hm, a bit risky maybe.

**Comment by crucialfelix** [comment]
This is a very simple bug fix. It doesn't require an army of developers to test it.

We have already spent way more time discussing it then the fix would have required.

File: SCClassLibrary/Common/Control/ServerStatus.sc [hunk 1]
```diff
@@ -55,8 +55,8 @@ ServerStatusWatcher {
 				failOSCFunc.free;
 			}, '/done', server.addr, argTemplate:['/notify', nil]).oneShot;
 
-			failOSCFunc = OSCFunc({|msg|
-				server.clientID = msg[2];
+			failOSCFunc = OSCFunc({|msg, time, replyAddr|
+				"failed to notify sclang by scserver. Please reboot the server '%'".format(server.name).warn;
```
Comment (telephon) [inline SCClassLibrary/Common/Control/ServerStatus.sc]: makes sense. I'll do that.

**Comment by telephon** [comment]
>This is a very simple bug fix. It doesn't require an army of developers to test it.

I meant this not for the current issue, but in general.

>We have already spent way more time discussing it then the fix would have required.

I didn't get it for a while …

**Pull request closed without merging**